### ISSUE TITLE
(suggestion)Some ideas i had for the mod

### ISSUE THREAD

**Issue by Wotzinator** [BluSunrize/ImmersiveEngineering#2920]
ive been playing with this mod for a long time now and there are a few things im missing in this mod from which i think they could fit and i hope that some of the following ideas will be added.

1.some kind of modifiable bow(or blunderbuss): i know immersive engineering is all about tech and it already has ranged weapons in it but there's just no early game weapon and an bow out of treated wood or steel,iron or something like that would be cool especially with a good model and and modifiers.and maybe even a modifier that turns it into a crossbow.

2.Armor : many tech mods are adding armor but most of the mare op or just ugly .since IE seems to have an tendency for great models and modifieable items it could be really great to see some armor in it besides the farady suit.my idea for this armor is to start as an simple armor made out of steel but with the folowing modifiers:
     1.adding stell plates to the armor would make it stronger and have a bulkier (and probably cooler) model.
     2.maybe some mod compability like an upgrade for forestry were you have an bekeeper upgrade wich protects you from bee stings.
     3.electrical spikes which could look similar to the ones on the shield and function similarly if you get hit      the attacker gets zapt (i should have a big cooldown).

3:chainsaw: an modular tool similar to the drill it should consume biodiesel and act like an normal axe with the difference that it can feel a whole tree and can have modifiers(besides that i can use the ones of the drill)like new blades(i think they are called like that)(the blades would determine how much damage it deos and how much durability it has).

4.skyhook rework: i like the skyhock but there are many things that i dont like about it .nearly any other tool/weapon has its own 3d model so i dont realy get why the skyhock doesnt .it dont works that good it often bugs out and the most time cables are out of range so i think it would be cool to have it like aome sort of  legend of zelda style grappling hook so that cables are never out of range.

5.vehicles: so i thought some sort of vehicle would be cool like min-carts(i know that already a mod exists that adds this but its not very good on lower end PC s and the models are a bit to detailed for my opinion) maybe some sort of flying machine like early  airplanes it would also be cool to have some water mobility especially with the 1.13 update in works.

6.explosives:first thing before its getting to the "boom" stuff i just wanted to say that obtaining gunpowder is a real pain so it would be nice to have another resource for the production of ammo and explosives or an easier way of getting gunpowder.so with that out of the way grenades more sort of TNT fireworks and so on would be extremely cool maybe with customize able effects.

i hope that you like these suggestions aaaand sorry for my bad englisch

**Comment by Saklad5** [comment]
As an alternative to a bow, you could have some crude sort of gun like a blunderbuss that happens to be mechanically similar to a bow. That would fit much better with the mod, I think. It could even shoot arbitrary items like Fallout 4’s Junk Jet.

Armor would be interesting. I already use the Reinforced Shield, since literally no other mod I know of has a shield. I especially like that it can be made to match Botania gear, which has possibly the best early progression around while being indispensable at all stages of the game. Still, similar armor would be nice. I don’t think the guns are currently good enough to justify use, but that might just be me.

**Comment by Wotzinator** [comment]
i just thaught that the bow would be a god idea because i nearly never have much gunpowder and also i thaught of a more mechanical 3d model but a blunderbuss is also a very cool idea .

to the armor i thaught that there may some different styles i know that i would be very difficult to do but some sort of samurai armor would be awsome.

also some sort of vehicle would be cool to

**Comment by Saklad5** [comment]
Couldn’t the early-game bow just be, you know, the Bow? I mean, there’s nothing wrong with it.

**Comment by Wotzinator** [comment]
anyways its the decision of blue sunrise

**Comment by Wotzinator** [comment]
but we dont even know if he ll add any of our ideas

**Comment by PrincessRTFM** [comment]
I know Blu (note the spelling, _no `e`_, he's emphasized this before) doesn't want to make the Faraday armor very protective, because the point is that you have to either get damage protection or tesla protection. The bow idea sounds interesting, but I would propose a recurve bow type model, if implemented. As for the guns not being good enough to justify use, I know that in 1.7.10 you could configure the damage done by each of the different shots for the revolver, and I'm pretty sure I saw something about configuring the railgun damage as well.

**Comment by Saklad5** [comment]
It’s not necessarily the fault of Immersive Engineering so much as it is the fault of Tinker’s Construct, if I’m being honest. I know I could up the damage, but it’s the lack of room for growth that is the real reason I don’t consider them worthwhile. By the time you have the infrastructure to deploy Immersive Engineering firearms, you can access weapons of similar effectiveness from other mods that can eventually become _far_ more powerful.

I don’t think there’s any way to change that without going far beyond the focus and spirit of Immersive Engineering, either. You might as well complain about all the beautiful multiblocks that are eventually outdone by fully-upgraded Thermal Expansion machines.

**Comment by Wotzinator** [comment]
to the farady suit thing it just was an idea.and im not an expert at bow desingnes so i dont know the differences.

**Comment by FerroO2000** [comment]
Coke oven advanced is under developement in IT

**Comment by Wotzinator** [comment]
ok that is cool ill remove it from the list

**Comment by HappyPineapple** [comment]
How bout a compound bow and I already suggested a saw and armor. No to saw but no activity on the armor.

**Comment by Flaxbeard** [comment]
I also have a more advanced coke oven in the works

**Comment by FerroO2000** [comment]
Lol
Is it over 50% of his developement?

**Comment by Flaxbeard** [comment]
Functionality is mostly finished (just need to deal with creosote + implement GUI), model/texture/animations are about half done

**Comment by Wotzinator** [comment]
flaxbeard are interested in adding more from this list in your addon im asking because it is very unlicly that all of my suggestions end up in a mod at all so having them in addons for the mod is also cool

**Comment by BluSunrize** [comment]
Guess I should go through the list quick:

1. I don't think a compound bow fits well with the style of IE. It's too slender, not clunky enough for IE's industrial style. Additionally, we have multiple weapons available, and I never wanted to make this a weapons mod in the first place.

2. Yeah I know. I've been wanting to add armor for a while. I'm waiting on Hazard to actually model it.

3. Something for woodcutting would be nice. Not sure if I'd make it a chainsaw, buzzsaw might look cooler tbh.

4. Skyhook gets a model when someone actually models it. If 3d modelling it was easy, we'd have already done it.

5. Nope. Not touching vehicles with a 10 foot pole. That's SkySom's problem.

6. Gunpower is craftable currently, IE adds the relevant components and means of processing them.

**Comment by Wotzinator** [comment]
ok then ill close this
